amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/PollingConfig.java

```diff
@@ -129,6 +131,14 @@ public void setIdleTimeBetweenReadsInMillis(long idleTimeBetweenReadsInMillis) {
         this.idleTimeBetweenReadsInMillis = idleTimeBetweenReadsInMillis;
     }
 
+    public void maxRecords(int maxRecords) {
```

Comment:
nit: setMaxRecords(...);

Reply:
The class has an annotation `@Accessors(fluent = true)`, so the getters and setters do not have the "get" and "set" prefix ([ref](https://projectlombok.org/features/experimental/Accessors)).

Reply:
@vincentvilo-aws this broke backwards compatibility. The method needed to return PollingConfig. See https://github.com/awslabs/amazon-kinesis-client/issues/1274 and https://github.com/awslabs/amazon-kinesis-client/pull/1275

Reply:
Thanks for catching this. I missed that the `chain` boolean by default is set to true when using the `@Accessors` annotation. I'll approve #1275 once the checks pass.